src/test/scala/uk/gov/hmrc/test/api/requests/SuppressionRulesRequests.scala

```diff
@@ -28,9 +28,9 @@ import uk.gov.hmrc.test.api.utils.{BaseRequests, TestData}
 
 object SuppressionRulesRequests extends ScalaDsl with EN with Eventually with Matchers with BaseRequests {
 
-  def postSuppressionData(json: String): StandaloneWSResponse = {
+  def postSuppressionData(json: String, id: String): StandaloneWSResponse = {
     val bearerToken = createBearerToken(enrolments = Seq("read:suppression-data"))
-    val baseUri     = s"$interestForecostingApiUrl/suppressions/1/suppression"
+    val baseUri     = s"$interestForecostingApiUrl/suppressions/" + id + "/suppression"
```

Comment:
maybe we can use string interpolator here like s"$interestForecostingApiUrl/suppressions/$id/suppression"